scripts/checkpatch.pl

```diff
@@ -3105,7 +3159,7 @@ sub process {
 					$comment = '/*';
 				} elsif ($realfile =~ /\.(c|dts|dtsi)$/) {
 					$comment = '//';
-				} elsif (($checklicenseline == 2) || $realfile =~ /\.(sh|pl|py|awk|tc)$/) {
+				} elsif (($checklicenseline == 2) || $realfile =~ /\.(sh|pl|py|awk|tc|yaml)$/) {
```

Comment:
This would introduce a license check against .yaml files in the Zephyr tree, right?

Reply:
Presumably.  AFAIK we were already requiring that, it just (maybe) wasn't being checked.